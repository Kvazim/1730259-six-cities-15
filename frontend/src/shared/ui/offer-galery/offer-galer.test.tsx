import { MemoizedOfferGalery } from './offer-galery';
import { render } from '@testing-library/react';

const testImages = ['img1', 'img2', 'img3', 'img4', 'img5', 'img6', 'img7', 'img8', 'img9', 'img10', 'img11'];

describe('OfferGalery', () => {
  it('should render correctly', () => {
    const {container} = render(<MemoizedOfferGalery images={testImages} />);
    expect(container).toMatchSnapshot();
  });
});
