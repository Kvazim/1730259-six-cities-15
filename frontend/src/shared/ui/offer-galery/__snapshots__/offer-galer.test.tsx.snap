// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`OfferGalery > should render correctly 1`] = `
<div>
  <div
    class="offer__gallery-container container"
  >
    <div
      class="offer__gallery"
    >
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img1"
        />
      </div>
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img2"
        />
      </div>
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img3"
        />
      </div>
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img4"
        />
      </div>
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img5"
        />
      </div>
      <div
        class="offer__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="offer__image"
          src="img6"
        />
      </div>
    </div>
  </div>
</div>
`;
